components/widget/match/summary/widget_match_summary_game_center.lua

```diff
@@ -19,7 +19,7 @@ local MatchSummaryGameCenter = Class.new(Widget)
 ---@return Widget?
 function MatchSummaryGameCenter:render()
 	return HtmlWidgets.Div{
-		classes = {'brkts-popup-body-element-vertical-centered'},
+		classes = {'brkts-popup-spaced'},
```

Comment:
probably needs testing if it doesn't break stuff somewhere
(e.g. on sc(2), ssg, wc if there is faction icons in some of the games or only on one side)

Reply:
Tested on a SC2 with a faction icon, no impact